components/table/test/table.visual-diff.js

```diff
@@ -1,6 +1,20 @@
 import puppeteer from 'puppeteer';
 import VisualDiff from '@brightspace-ui/visual-diff';
 
+async function getRect(page, id) {
+	return await page.$eval('d2l-test-table-visual-diff', (wrapper, id) => {
+		const elem = wrapper.shadowRoot.querySelector(`#${id}`);
+		const leftMargin = (elem.offsetLeft < 10 ? 0 : 10);
+		const topMargin = (elem.offsetTop < 10 ? 0 : 10);
+		return {
+			x: elem.offsetLeft - leftMargin,
+			y: elem.offsetTop - topMargin,
+			width: elem.offsetWidth + (leftMargin * 2),
+			height: elem.offsetHeight + (topMargin * 2)
+		};
+	}, id);
+}
+
 describe('d2l-table', () => {
```

Comment:
Just some refactoring in here to deal with the sticky tests coming in here and things being in shadow roots. The tests themselves haven't changed.